lib/cleanup.js

```diff
@@ -18,6 +18,19 @@ module.exports = {
             if (err) {
               reject(err);
             } else {
+              if (this.serverless.service.package.individually) {
```

Comment:
thanks! also #55 looks similar to this PR but it also has readme improvements. would you review that one as well please? 🍰